### ISSUE TITLE
Ore and Plants Bags are broken

### ISSUE THREAD

**Issue by Vermidia** [space-wizards/space-station-14#27208]
## Description

They can no longer do individual or area pickups.

**Reproduction**

Try to use a plant bag on plants or an ore bag on ores

**Screenshots**

**Additional context**

Fails on UseDelay.TryResetDelay, though unlike the Bible one I'm not sure exactly why here.

**Comment by UbaserB** [comment]
Had this happen to me ingame

**Comment by Tayrtahn** [comment]
Turns out Server's StorageSystem and SharedStorageSystem are both using UseDelay for separate purposes (UI opening sound spam prevention and quick/area pickup cooldown).

**Comment by Erisfiregamer1** [comment]
Still happening, I can't area pickup or quick pickup with plant bags. Was this behaviour removed?

**Comment by Vermidia** [comment]
I think it's still bugged and not intended to be removed.

**Comment by Erisfiregamer1** [comment]
Yeah, still bugged. Reopen, linked PR did not fix

**Comment by Tayrtahn** [comment]
Yeah, I actually only tested it with trash bags and assumed it fixed ore bags and plant bags too. Had a feeling that would come back to bite me. 

Anyway, the issue is that the bags don't have a UseDelay component, and Storage needs one for the insert cooldown. It'd be easy enough to add them to the prototypes, but I suspect that this issue will come up again, since it's not obvious that Storage requires UseDelay to function. I might modify UseDelay.SetLength to add the component if the entity doesn't have it.
